From 5f1f38d880a135082abe9a93b7bda46214e8aeeb Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Thu, 1 Jun 2017 15:09:07 +0500
Subject: [PATCH] layersvt: obey CMAKE_INSTALL_LIBDIR

Hardcoding to specific paths like install_staging
is not a very good idea when you are trying to
make the system flexible.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 layersvt/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/layersvt/CMakeLists.txt b/layersvt/CMakeLists.txt
index 62f1ca95e..3b7d016e9 100644
--- a/layersvt/CMakeLists.txt
+++ b/layersvt/CMakeLists.txt
@@ -99,7 +99,7 @@ else()
     target_link_Libraries(VkLayer_${target} VkLayer_utilsvt)
     add_dependencies(VkLayer_${target} generate_helper_files generate_api_cpp generate_api_h VkLayer_utilsvt)
     set_target_properties(VkLayer_${target} PROPERTIES LINK_FLAGS "-Wl,-Bsymbolic")
-    install(TARGETS VkLayer_${target} DESTINATION ${PROJECT_BINARY_DIR}/install_staging)
+    install(TARGETS VkLayer_${target} DESTINATION ${CMAKE_INSTALL_LIBDIR})
     endmacro()
 endif()
 
@@ -147,7 +147,7 @@ if (WIN32)
     add_library(VkLayer_utilsvt STATIC ../layers/vk_layer_config.cpp ../layers/vk_layer_extension_utils.cpp ../layers/vk_layer_utils.cpp ../layers/vk_format_utils.cpp)
 else()
     add_library(VkLayer_utilsvt SHARED ../layers/vk_layer_config.cpp ../layers/vk_layer_extension_utils.cpp ../layers/vk_layer_utils.cpp ../layers/vk_format_utils.cpp)
-    install(TARGETS VkLayer_utilsvt DESTINATION ${PROJECT_BINARY_DIR}/install_staging)
+    install(TARGETS VkLayer_utilsvt DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()
 add_dependencies(VkLayer_utilsvt generate_helper_files)
 
-- 
2.11.1

